packages/toolkit/form-field/form-field-wrapper.scss

```
@@ -144,7 +148,8 @@
   :host-context(:root:not(.dark)) {
     --_field-clr-primary: #007bc7;
     --_field-clr-danger: #db1818;
-    --_field-clr-warning: #f59e0b;
+    // Mirror the light-mode :host default (amber-700) for AA-on-white contrast.
```

Stylelint (17.7.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)